src/Sylius/Bundle/ApiBundle/CommandHandler/PickupCartHandler.php

```diff
@@ -98,6 +104,8 @@ public function __invoke(PickupCart $pickupCart)
 
         $this->orderManager->persist($cart);
 
+        $this->session->set('cart_token', $cart->getTokenValue());
```

Comment:
I guess that we shouldn't use session in handler

Reply:
I could skip the conversation, but did we decided that we still want to use the session here? Is it used anywhere?

Reply:
Sorry, i must have forgote to delete this line 😃